### ISSUE TITLE
support Django 1.10

### ISSUE THREAD

**Issue by kelseyq** [SexualHealthInnovations/callisto-core#105]
1.10 final is coming out [sometime in August](https://www.djangoproject.com/weblog/2016/jul/18/security-releases/)

**Comment by keisetsu** [comment]
Released [today](https://www.djangoproject.com/weblog/2016/aug/01/django-110-released/)!

**Comment by sunahsuh** [comment]
Confirmed a fresh install of the requirements file with Django 1.10 (the current default) is broken, due to this downstream issue: https://github.com/django-polymorphic/django-polymorphic/issues/225

Tests passed with Django 1.10 when I pointed the django-polymorphic requirement to the repo's master, though, so things should run once they bump the version in pip.

I also pinned the requirements file to django<1.10 and ran with deprecation warnings on, and the only warning for 1.10 that comes up is:
```
/Users/sunahsuh/.virtualenvs/callisto/lib/python3.5/site-packages/django/conf/__init__.py:129: RemovedInDjango110Warning: Session verification will become mandatory in Django 1.10.
Please add 'django.contrib.auth.middleware.SessionAuthenticationMiddleware' to your MIDDLEWARE_CLASSES setting when you are ready to opt-in after reading the upgrade considerations
in the 1.8 release notes.
  RemovedInDjango110Warning
```
(There are a number of Django 2.0 deprecation warnings, but that's being released 12/17)
The deprecation note is here: https://docs.djangoproject.com/en/1.10/topics/auth/default/#session-invalidation-on-password-change -- basically, the default is now to invalidate every other active session for a user on password change, which I suspect will not cause issues.

Perusing the release notes, it doesn't look like the project's likely to run into other compatibility issues, but I'm still getting to know the codebase.
